eth-balance/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+#[rustfmt::skip]
+pub mod abi;
+#[rustfmt::skip]
+pub mod pb;
+
+use num_bigint;
+use substreams::{store, Hex};
+use substreams::scalar::BigInt;
+use substreams_ethereum::{pb::eth as pbeth};
+use substreams::store::{StoreSetRaw, StoreSet, StoreNew};
+use pb::eth_balance::v1::{EthBalanceChanges, EthBalanceChange};
+
+#[substreams::handlers::map]
+fn map_block_to_balance_changes(
```

Comment:
Any particular reason of having a mapping module? Why not aggregate directly in the store module?

Reply:
No, I just wanted to build a substreams from scratch that will include all the available mapping kinds, I will remove it now and aggregate it directly in the store module